tests/e2e/lib/apps.go

```diff
@@ -36,8 +32,9 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/serializer/yaml"
 	"k8s.io/apimachinery/pkg/selection"
 	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/apimachinery/pkg/util/wait"
+	apiwait "k8s.io/apimachinery/pkg/util/wait"
```

Comment:
Please undo this change, I believe it is unnecessary

Reply:
Correct. I did change it as I previously had also my own wait for service function and did not want to mix two different wait names, however it's not required now. Will undo.